sdk/storage/inc/datalake/protocol/datalake_rest_client.hpp

```diff
@@ -15,6 +14,12 @@
 #include <string>
 #include <vector>
 
+#pragma warning(push)
+#pragma warning(disable: 4505) // Unreferenced local function.
+#pragma warning(disable: 28204)
+#pragma warning(disable: 28020)
```

Comment:
There are windows-specific?

Reply:
MSVC specific. I already moved them to a more general place and added comments.